resources/creatives/models/campaign_item.yml

```diff
@@ -0,0 +1,38 @@
+allOf:
+  - $ref: "../../../shared/models/lob_base.yml"
+  - $ref: "../../campaigns/models/campaign_writable.yml"
+
+  - type: object
+
+    required:
+      - id
+      - name
+      - description
+      - schedule_type
+      - use_type
+      - is_draft
+      - creatives
+      - auto_cancel_if_ncoa
+      - date_created
+      - date_modified
+      - object
+
+    properties:
+      id:
+        $ref: "../../campaigns/attributes/cmp_id.yml"
+
+      is_draft:
+        description: Whether or not the campaign is still a draft.
+        type: boolean
+        default: true
+
+      creatives:
+        description: An array of creatives that have been associated with this campaign.
+        type: array
+
+      object:
+        type: string
+        description: Value is resource type.
+        enum:
+          - campaign
+        default: campaign
```

Comment:
```suggestion
  - $ref: "../../campaigns/models/campaign_writable.yml"

  - type: object

    required:
      - id
      - name
      - description
      - schedule_type
      - use_type
      - is_draft
      - creatives
      - auto_cancel_if_ncoa
      - date_created
      - date_modified
      - object

    properties:
      id:
        $ref: "../../campaigns/attributes/cmp_id.yml"

      is_draft:
        description: Whether or not the campaign is still a draft.
        type: boolean
        default: true

      creatives:
        description: An array of creatives that have been associated with this campaign.
        type: array

      object:
        type: string
        description: Value is resource type.
        enum:
          - campaign
        default: campaign

      date_created:
        $ref: "../../../shared/attributes/timestamps.yml#/date_created"

      date_modified:
        $ref: "../../../shared/attributes/timestamps.yml#/date_modified"

      deleted:
        $ref: "../../../shared/attributes/deleted.yml"
```

Because referencing `lob_base` here results in a recursive reference (it's used further up the chain)
<img width="886" alt="image" src="https://user-images.githubusercontent.com/45194378/183616832-9561df04-3ce9-4b1c-b076-83c6e0e29ce8.png">
Removed the ref and added the fields to `properties`